docs/tutorial/geo_spark_core_python.md

```diff
@@ -86,23 +86,23 @@ uses findspark Python package to upload jar files to executor and nodes. To avoi
 Please use command below
  
 ```bash
-pip install geo_pyspark
+pip install geo-pyspark
 ```
 
 ### Installing from wheel file
 
 
 ```bash
 
-pipenv run python -m pip install dist/geo_pyspark-0.3.0-py3-none-any.whl
+pipenv run python -m pip install dist/geo_pyspark-1.3.0-py3-none-any.whl
```

Comment:
You use geo_pyspark at some places and geo-pyspark at some other places. Can you make them consistent?

Reply:
I still do not know why PyPi gave the project name as geo-pyspark instead of geo_pyspark. That is the reason behind different names in pip install geo-pyspark and pip install from wheel file. Everywhere I am using geo_pyspark, so it is trange to me why did PyPi do that.